unicorecmsmnm/templates/base.pt

```diff
@@ -13,21 +13,28 @@
     <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;" />
     <meta name="description" content=""/>
     <meta name="keywords" content="malaria, health"/>
-    <link href="${request.static_url('cms:static/css/style.css')}?v=20141014-2" rel="stylesheet" type="text/css" />
+    <link href="${request.static_url('cms:static/css/style.css')}" rel="stylesheet" type="text/css" />
```

Comment:
This needs version revving. Whack a `?v=20150128` on the end.

---

Browsers cache stylesheets.
That's great because they take the hit of downloading the stylesheet once.
That's terrible because when we push up a new stylesheet with the same name, the browser says "nah, it's cool, I've got that file already."